src/fetch_receipts.py

```diff
@@ -174,4 +204,5 @@ def run(target, limit=None):
     args = parser.parse_args()
 
     # run
-    run(args.target, args.limit)
+    result = run(args.target, args.limit)
+    print_report(result)
```

Comment:
I tried putting the `print_report()` inside `run()`, but then the script hangs after downloading the files.